ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1165,3 +1165,4 @@ admin.message.queue.tab.scheduled = Scheduled
 admin.message.queue.tab.due = Due
 admin.message.queue.tab.muted.future = Will not send
 admin.message.queue.tab.muted.past = Did not send
+already\ taken = is already taken.
```

Comment:
Please use the lowercase dot-separated key format, as indicated in https://github.com/medic/medic-docs/blob/master/development/translations.md.